projects/packages/search/src/customberg/components/sidebar/sidebar-description.jsx

```diff
@@ -16,7 +16,7 @@ export default function SidebarDescription() {
 			<div>
 				<p>
 					{ __(
-						'Jetpack Instant Search will allow your visitors to get search results as soon as they start typing. Customize this experience to offer better results that match your site.',
+						'Jetpack Search will allow your visitors to get search results as soon as they start typing. Customize this experience to offer better results that match your site.',
```

Comment:
This change means the `Jetpack Search` will only include `Instant Search` in the future because we plan to remove `Classic Search`, am I right?

Reply:
Correct —  we'd like to reframe "instant search" as a feature and move away from branding our product as "Instant Search".